spec/serializers/automation_field_serializer_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe DiscourseAutomation::FieldSerializer do
+  let(:automation) { DiscourseAutomation::Automation.create!(script: 'gift_exchange') }
+
+  context "with a TL0 user seen as anonymous" do
+  end
+end
+
+
+    # attributes :id, :component, :name, :metadata, :placeholders
```

Comment:
Needed?

Reply:
no that something I wanted to test but have postponed for now, removing